### DISCUSSION TITLE
Migrating react query 3.6 to TanStack query 4.20

### DISCUSSION THREAD

**Discussion by rickvian** [TanStack/query#4758] [Q&A] [answered]
Hi, i was planning to bump my production code from "react-query": "3.6.0" to latest 
"react-query" : "4.20" at the time i wrote this.

we have lots of unit tests and  using "msw": "0.38.0" as well

is there any potential issue i need to aware / took care of?

**Comment by TkDodo** [comment, marked as answer]
It need to be `@tanstack/react-query`. Migration guide is here: https://tanstack.com/query/v4/docs/react/guides/migrating-to-react-query-4

**Reply by rickvian** [reply]
Thank you!
